test/val/val_ext_inst_debug_test.cpp

```diff
@@ -1547,41 +1547,6 @@ OpExtension "SPV_KHR_non_semantic_info"
                         "integer scalar type"));
 }
 
-TEST_F(ValidateVulkan100DebugInfo, DebugTypeArrayFailComponentCountZero) {
```

Comment:
I feel like it would be better to just rename the test and ensure it passes than remove the coverage altogether.